annotation-processor/src/test/java/org/hibernate/validator/ap/testmodel/annotationparameters/InvalidGroupSequenceParameters.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.ap.testmodel.annotationparameters;
+
+import javax.validation.GroupSequence;
+import javax.validation.constraints.AssertFalse;
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * @author Marko Bekhta
+ */
+public class InvalidGroupSequenceParameters {
+
+	/**
+	 * Case 1: class present in a list of groups
+	 */
+	public static class Case1 {
+		public interface Group1 {
+		}
+
+		public interface Group2 {
+		}
+
+		@GroupSequence(value = Group2.class)
```

Comment:
Are `Group2`, `Group3` and `YetOtherBean` actually needed for this case? All tests pass if I comment them out, so they could go away it seems?